ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/individual-journey/services/blob-storage.service.ts

```diff
@@ -14,6 +14,10 @@ export class BlobStorageService {
   }
 
   getVideoUrl(videoName: string): string {
-    return this.baseVideoUrl + this.separator + videoName;
+    if (!!videoName) {
```

Comment:
if not not videoname than videoname is.

Reply:
the operand !! is  eliminate null, undefined, 0, empty value , if value is one of these return false , otherwise true.